<launch>

  <param name="simtrack/model_path" value="$(find vision)/../data/object_models" />
  <rosparam command="load" file="$(find vision)/config/demo_objects.yaml"/>

  <rosparam command="load" file="$(find vision)/config/parameters.yaml"/>
  <param name="simtrack/use_compressed_streams" value="true" />
  <param name="simtrack/tracker/color_only_mode" value="false" />

  <node name="tracker" pkg="simtrack_nodes" type="multi_rigid_tracker_node" output="screen">
  <remap from="depth/compressedDepth" to="/head_mount_kinect/depth_registered/image_rect_raw/compressedDepth" />
  <remap from="rgb/compressed" to="/head_mount_kinect/rgb/image_rect_color/compressed" />
  <remap from="rgb_info" to="/head_mount_kinect/rgb/camera_info" />
  </node>
  
  <node name="detector" pkg="simtrack_nodes" type="multi_rigid_detector_node" output="screen">
  <remap from="rgb/compressed" to="/head_mount_kinect/rgb/image_rect_color/compressed" />
  <remap from="rgb_info" to="/head_mount_kinect/rgb/camera_info" />
  </node>
  
</launch>
